www/components/Nav/index.tsx

```diff
@@ -243,10 +243,16 @@ const Nav = (props: Props) => {
               </div>
               <div className="hidden lg:flex items-center sm:space-x-3">
                 <a href="https://github.com/supabase/supabase" target="_blank">
-                  <Button type="default" icon={<IconStar />}>
+                  <Button
+                    className="sbui-btn sbui-btn-text sbui-btn--tiny sbui-btn--text-align-center"
+                    icon={<IconStar />}
+                  >
                     Star us on GitHub
                   </Button>
                 </a>
+                <a href="https://app.supabase.io/">
+                  <Button type="default">Sign in</Button>
+                </a>
                 <a href="https://app.supabase.io/">
                   <Button>Start your project</Button>
                 </a>
```

Comment:
Shouldn't we use `Link` from Nextjs?

Reply:
Yes, we should use but previously this component is using anchors for the other 2 links, so to maintain consistency I did not use Link. I can update that if that sounds good.

Reply:
Link is better for "internal" links (prevents full reload), and anchor tags for external